home-work.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"unicode/utf8"
+)
+
+func average(arr []int) float64 { // Task 1
+	sum := 0
+	for _, a := range arr {
+		sum += a
+	}
+	return float64(sum) / float64(len(arr))
+}
+
+func max(words []string) string { // Task 2.1
+	word := ""
+	for _, a := range words {
+		if utf8.RuneCountInString(a) > utf8.RuneCountInString(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func reverse(arr []int64) []int64 { // Task 2.2
+	ar := append(arr, 0)
+	for i, j := 0, len(ar) - 1; i < len(ar) / 2; i, j = i + 1, j - 1 {
+		ar[i], ar[j] = ar[j], ar[i]
+	}
+	return ar[1:]
+}
+
+func printSorted(m map[int]string) { // Task 3
+	var tmp []int
+	var res []string
+	for k := range m {
+		tmp = append(tmp, k)
+	}
+	sort.Ints(tmp)
+	for _, v := range tmp {
+		res = append(res, m[v])
```

Comment:
There is no need to create `res` slice. You should just print map values.

Reply:
I thought it was necessary to make an identical conclusion as in the presentation))